ui/src/layouts/Baseline.vue

```diff
@@ -66,14 +78,34 @@
     </v-app>
 </template>
 
-<script>
-import { mapGetters, mapState } from 'vuex'
+<script setup>
+import { useTheme } from 'vuetify'
```

Comment:
Not sure if these would be better placed in `App.vue`...

Reply:
I think here is sensible as all layouts extend this `Baseline` anyway

Reply:
Ok, great.